src/utils/hook.ts

```diff
@@ -367,7 +367,7 @@ export function useMediaQuery(query: string): boolean {
 }
 
 export const MOBILE_DEVICE_MAX_WIDTH = 750
-export const useIsMobile = () => useMediaQuery(`(max-width: ${MOBILE_DEVICE_MAX_WIDTH}px)`)
+export const useIsMobile = (max?: number) => useMediaQuery(`(max-width: ${max ?? MOBILE_DEVICE_MAX_WIDTH}px)`)
```

Comment:
`useIsMobile` is actually a JavaScript version of the 750px breakpoint. It should be a fixed value that is equal to the breakpoint defined in CSS, so it should not accept a variable parameter.